Fiction notes/GDL_PorosityDistribution_collector.md

```diff
@@ -1,6 +1,14 @@
 <!-- <script type="text/javascript" src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=default"></script> -->
 
-### Cross-Subject
+HZQ: 2019/4/23, "DOCS: add arrangement structure"
```

Comment:
Here use "content" to guide reading, meanwhile, include fragment idea from single article reading